src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -13,638 +13,637 @@
 !  1 - Unrecoverable system or logical error.
 !  2 - All provided observations are invalid.  Cannot create NetCDF IODA file.
 
-
 program gnssro_bufr2ioda2
-use netcdf
-implicit none
+   use, intrinsic :: iso_fortran_env
+   use netcdf
+   implicit none
 
-integer, parameter :: i_kind  = selected_int_kind(8)    !4
-integer, parameter :: i_64    = selected_int_kind(10)   !8
-integer, parameter :: r_kind  = selected_real_kind(15)  !8
+!integer, parameter :: i_kind  = selected_int_kind(8)    !4
+!integer, parameter :: int64    = selected_int_kind(10)   !8
+!integer, parameter :: real64  = selected_real_kind(15)  !8
```

Comment:
are these commented-out lines needed?

Reply:
@fmahebert That was stripped out of the new version of this file. I left it in the old one until I can ascertain why it was failing. Turned out there were line contrinuations, "&", in the middle of strings. Thanks for the catch.